types/parse/index.d.ts

```diff
@@ -1112,6 +1113,16 @@ namespace Parse {
         function track(name: string, dimensions: any): Promise<any>;
     }
 
+    /**
+     * Provides utility functions for working with Anonymously logged-in users.
+     */
+    namespace AnonymousUtils  {
+        function init(): void;
```

Comment:
I can't find any reference to the `init()` function in the [source code](https://github.com/parse-community/Parse-SDK-JS/blob/master/src/AnonymousUtils.js). Should we remove this?
```suggestion
```